docs/src/content/v1-migration/forEachObj.md

```diff
@@ -0,0 +1,70 @@
+# Typing
+
+The `indexed` variant was removed; the base implementation takes the same
+parameters. If you are using `indexed` you can simply remove it without any
+other changes.
+
+Symbol keys are no longer passed to the callback function, and their values are
+also excluded from the typing.
+
+Number keys are cast to strings when passed to the predicate.
+
+# Runtime
+
+The callback function now takes 2 additional parameters: `key` - The key of the
+current entry being processed in data, and `data` - the object the function was
+called upon (similar to the signature for callbacks in `Array.prototype`).
+
+If you are using a function reference for the predicate (and not an inline arrow
+function), and that function accepts more than one param you might run into
+compile-time (or run-time!) issues because of the extra params being sent on
+each invocation of the function. We highly recommend using [unicorn/no-array-callback-reference](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-callback-reference.md)
+to warn against these issues.
+
+## Examples
+
+### Indexed variant removed
+
+```ts
+// Was
+forEachObj.indexed({ a: 1, b: 2 }, (value, key) => {
+  console.log(`${key} => ${value}`);
+});
+
+// Now
+forEachObj({ a: 1, b: 2 }, (value, key) => {
+  console.log(`${key} => ${value}`);
+});
+```
+
+### Symbol Keys
```

Comment:
**nit:** `Keys` -> `keys`

Reply:
(our headings are in sentence case)

Reply:
lmao this is an extremely tiny nit, i will probably just give it a copyediting pass when they're all written

Reply:
These become repetitive to write and are quite draining for me, I'm happy that another person is reading them and getting a feel for them so they stay consistent.